packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/components",
-	"version": "2.3.0",
+	"version": "2.4.0",
```

Comment:
This release includes a breaking change. Wouldn't it be 3.0.0? (I see the previous release is wrong too)

Reply:
Fair enough, working on `@wordpress/components` numbed by SEMVER